src/responsibleai/rai_analyse/create_score_card.py

```diff
@@ -0,0 +1,228 @@
+import argparse
+import json
+import logging
+import os
+import re
+
+from azureml.core import Run
+from responsibleai import __version__ as responsibleai_version
+
+from _rai_insight_data import RaiInsightData, PdfDataGen
+
+from generate_pdf import RegressionComponents, ClassificationComponents, to_pdf, get_full_html
+
+from constants import DashboardInfo, PropertyKeyValues, RAIToolType
+from rai_component_utilities import load_dashboard_info_file
```

Comment:
Or maybe not on having the separate directory. Not unless we publish some PyPI package as a 'helper' which is complication which we don't need right now.

Reply:
Makes sense after your earlier comment.